sheepdog/transactions/upload/sub_entities.py

```diff
@@ -735,3 +691,32 @@ def _create_index(self, **kwargs):
 
     def _version_index(self, **kwargs):
         return self.transaction.indexd.add_version(**kwargs)
+
+    def _should_version_node(self, node):
```

Comment:
so this prevents versioning in indexd? is the node updated on the graph?

Reply:
This prevents calling node_recreate which deletes current node and sets action to `version` is then used to update indexd. All nodes properties always gets updated cos of the way the code is structured. This determines when versioning when versioning should happen

Reply:
👍